src/hlstrafe.cpp

```diff
@@ -498,6 +498,29 @@ namespace HLStrafe
 		avec[1] = wishvel[1];
 	}
 
+	void UpdateLookAtViewangle(const PlayerData& player, CurrentState& curState)
+	{
+		curState.TargetYawLookAtOrigin[0] += curState.Parameters.Parameters.LookAt.X;
+		curState.TargetYawLookAtOrigin[1] += curState.Parameters.Parameters.LookAt.Y;
+		curState.TargetYawLookAtOrigin[2] += curState.Parameters.Parameters.LookAt.Z;
```

Comment:
I wonder if instead of this in-place update, it would be better to do the equivalent of `const auto look_at_origin = curState.TargetYawLookAtOrigin + LookAtXYZ`. This way it's no longer necessary to refresh TargetYawLookAtOrigin every frame, which could be useful in some other code using HLStrafe.